packages/core/src/client.ts

```diff
@@ -2712,6 +2714,17 @@ export class MedplumClient extends EventTarget {
     return this.request('POST', url, options);
   }
 
+  /**
+   * Returns the key value client.
+   * @returns The key value client.
+   */
+  get keyValue(): MedplumKeyValueClient {
```

Comment:
Love to see it. Just the right amount of magic here.

I think we can also lazy load the `./keyvalue` module here but not blocking this PR.

We should definitely do this to the `FHIRcast` client and `SubscriptionManager` too I think. If we actually bundle them as (a) separate file(s) potentially could reduce bundle size and initial load times a lot for those who are not actually using any of those fancy features.